source/include/cfg/cs_StaticConfig.h.in

```diff
@@ -18,7 +18,8 @@
 extern "C" {
 #endif
 
-const uintptr_t g_RAM_MICROAPP_END = (@RAM_MICROAPP_BASE@ + @RAM_MICROAPP_AMOUNT@);
+// Microapp ram is located before IPC ram.
+const uintptr_t g_RAM_MICROAPP_END = @RAM_R1_BASE@ + @RAM_APPLICATION_AMOUNT@ - @RAM_BLUENET_IPC_LENGTH@;
```

Comment:
This seems wrong... I suppose the amount of ram for bluenet is `RAM_APPLICATION_AMOUNT` and this should exclude the amount of ram for the microapp. Hence, `g_RAM_MICROAPP_END` should be at:

```
const uintptr_t g_RAM_MICROAPP_END = @RAM_R1_BASE@ + @RAM_APPLICATION_AMOUNT@ + @RAM_MICROAPP_AMOUNT@;
```

Reply:
That, or rename, `@RAM_APPLICATION_AMOUNT@` to something like `@RAM_TOTAL_AMOUNT@` (or something better).

Reply:
I left the meaning of `RAM_APPLICATION_AMOUNT` as it was, it already included the IPC ram.